### ISSUE TITLE
Strange behavior of side menu 6.0.2

### ISSUE THREAD

**Issue by 22Round** [jonkykong/SideMenu#498]
**I have read the [guidelines for contributing](https://github.com/jonkykong/SideMenu/blob/master/.github/CONTRIBUTING.md) and I understand**
- [x] My issue is happening in the **latest version** of SideMenu (older versions are no longer maintained).
- [x] My issue was **not** solved in the [README](https://github.com/jonkykong/SideMenu/blob/master/README.md).
- [x] My issue can **not** be answered on [stackoverflow.com](stackoverflow.com).
- [x] My issue is **not** a request for new functionality that I am unwilling to build and contribute with a pull request.
- [x] My issue **is** reproducible in the [demo project](https://github.com/jonkykong/SideMenu/tree/master/Example).

**Describe the bug**
updated pod to latest 6.0.2 version.

**To Reproduce**
my implementation is programatic.

```
let leftMenu = LeftMenuViewController()
var settings:SideMenuSettings = SideMenuSettings()
settings.menuWidth = ScreenSize.width - 46
        
let leftMenuNavigationController = UISideMenuNavigationController(rootViewController: leftMenu, settings: settings)
SideMenuManager.default.leftMenuNavigationController = leftMenuNavigationController
```

and then when I want to present left menu

``` 
if let leftMenu = SideMenuManager.default.leftMenuNavigationController {
       present(leftMenu, animated: true, completion: nil)
 }
```

**Expected behavior**
present menu from left side, main content slide to right

**Screenshots**
![buggif](https://media.giphy.com/media/UvDbeGY2kTstMPDbv6/giphy.gif)

**Additional context**
version 5.0.3 worked as expected

**Comment by rgmorrison** [comment]
I'm also seeing this exact same behavior

**Comment by SunsetWan** [comment]
I have met the same issue too when upgrading to 6.0.

**Comment by limadeveloper** [comment]
Guys, for me the latest version 6.0.2 works perfectly.
I'm using Xcode 11 beta 3, iOS 13.
Here are my implementation: 
```swift
import SideMenu

final class MenuManager {

  typealias SideMenuNavigation = UISideMenuNavigationController
  
  // MARK: - Properties
  static let shared = MenuManager()
  
  private var target: UIViewController?
  private var lefMenuNavigation: UISideMenuNavigationController?

  private var menuSettings: SideMenuSettings {
    let presentationStyle: SideMenuPresentationStyle = .menuSlideIn
    presentationStyle.backgroundColor = .clear

    var settings = SideMenuSettings()
    settings.presentationStyle = presentationStyle
    settings.menuWidth = UIScreen.main.bounds.width * 0.77
    settings.pushStyle = .popWhenPossible
    settings.alwaysAnimate = true
    settings.usingSpringWithDamping = 0.9
    settings.blurEffectStyle = .regular

    return settings
  }

  // MARK: - Public Methods
  func setupLeftMenu(withTarget target: UIViewController) {
    lefMenuNavigation = UIStoryboard.leftMenuNavigationController
    lefMenuNavigation?.settings = menuSettings

    SideMenuManager.default.leftMenuNavigationController = lefMenuNavigation
    SideMenuManager.default.addPanGestureToPresent(toView: target.view)

    self.target = target
    
    setupMenuButton()
    
    guard let navigationBar = target.navigationController?.navigationBar else { return }

    SideMenuManager.default.addPanGestureToPresent(toView: navigationBar)
  }
  
  func dismissLeftMenu(completion: LMCompletion.EmptyBlock? = nil) {
    lefMenuNavigation?.dismiss(animated: true, completion: completion)
  }
  
  // MARK: - Private Methods
  private func setupMenuButton() {
    let button = UIButton()
    button.setIcon(.sideMenu, size: 24, weight: .semibold)
    button.addTarget(self, action: #selector(showMenu(sender:)), for: .touchUpInside)
    target?.navigationItem.leftBarButtonItem = UIBarButtonItem(customView: button)
  }
  
  @objc
  private func showMenu(sender: UIBarButtonItem) {
    guard let menu = SideMenuManager.default.leftMenuNavigationController else { return }
    target?.present(menu, animated: true, completion: nil)
  }
}
```
I hope it can help you

**Comment by 22Round** [comment]
@limadeveloper as I assumed  you are using ***StoryBoard***

**Comment by Hardiktspaceo** [comment]
Hello Team,

I also face same issue when upgrading to 6.0.2. latest version. Please update us how to solve that.

**Comment by jonkykong** [comment]
Thank you for reporting. This is fixed in 6.0.3.

**Comment by Amolpokaleitgs** [comment]
I facing the same issue for ios12 and iOS 13 as well, using latest pods. Following line of code that I have used for showing the slider using slide gesture 

`   let mainStoryboard = UIStoryboard(name: Constant.StoryBoardIdentifier.main, bundle: nil)
        if let controller = mainStoryboard.instantiateViewController(withIdentifier: "MenuViewController") as? MenuViewController {
            if SideMenuManager.default.rightMenuNavigationController == nil {
                let rightMenuNavigationController = SideMenuNavigationController(rootViewController: controller)
                SideMenuManager.default.rightMenuNavigationController = rightMenuNavigationController
                rightMenuNavigationController.statusBarEndAlpha = 0
                rightMenuNavigationController.menuWidth = min(view.frame.width, view.frame.height) * CGFloat(0.75)
                rightMenuNavigationController.presentDuration = 0.0
                SideMenuManager.default.addScreenEdgePanGesturesToPresent(toView: view)
                }
        }`

When we open menu using gesture 
![IMG_5037451F7177-1](https://user-images.githubusercontent.com/48118963/69955121-f3b4b880-1522-11ea-8e10-c8190f5c968c.jpeg)


after closing the slider menu 
![IMG_618C46DBD4CA-1](https://user-images.githubusercontent.com/48118963/69955120-f3b4b880-1522-11ea-9a59-1a83c6eaad29.jpeg)

Could you please revert back ASAP.
